src/components/CountrySelector/PoACountrySelector.tsx

```diff
@@ -22,6 +21,15 @@ export type PoaProps = {
   StepComponentBaseProps
 
 class CountrySelection extends CountrySelectionBase {
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      showNoResultsError: false,
+      alwaysShowEmptyMessage: true,
```

Comment:
I would only change what you actually need to change, instead of overriding the whole state. `this.state.alwaysShowEmptyMessage = true` should be enough.